ESMF/Aerosol_GridComp/Aerosol_GridComp.F90

```diff
@@ -43,6 +47,9 @@ subroutine SetServices(gc, rc)
         call ESMF_UserCompSetInternalState(gc, internal_name, wrap, status)
         VERIFY_(status)
 
+        ! add children
+        gocart = MAPL_AddChild(gc, name='GOCART2G', ss=GOCART2G_SetServices, __RC__)
+
```

Comment:
OK - this one surprises me.  Didn't we add gocart as a child a long while back?  I'm clearly confused - just want to understand what I'm missing.

Reply:
@tclune - Aerosol_GridComp is also not used/needed for the NOAA UFS integration

Reply:
@rmontuoro if it is not being used, then can it be removed? It is not being used in GEOS.

Reply:
Aerosol_GridComp is likely used in GEOS for the carbon and CFC components in legacy GOCART.

Reply:
@gmao-esherman, This is incorrect, this component was introduced as the original "root" component for the effort to allow GOCART2G to be used in the UFS. The development has now moved away from that.

Reply:
My mistake. I was referring to Aero_GridComp.

Reply:
I take that back. Aerosol_GridComp *is* needed to initialize  and run MAPL Cap in the NUOPC/UFS cap:

```
  use Aerosol_GridComp_mod, only : &
    aerosol_routine_SS          => SetServices
  [...]
  is % maplCap = MAPL_Cap("MAPL Driver", aerosol_routine_SS, cap_options=maplCapOptions, _RC)
```

@tclune - Are additional changes required to correctly set up the MAPL component hierarchy?

Reply:
This aspect is slightly unstable in the midst of my other recent refactorings and thus a bit hard to assess from just a quick glance at the code.    Generally if anything is wrong wrt to the MAPL hierarchy, things will break immediately.   So "working for you" is a sufficient indicator, and all that I can offer at the moment.   Of course if you do see a problem, we'll jump in immediately to diagnose.